docs/en/alab/project_creation.md

```diff
@@ -0,0 +1,81 @@
+
+In the Project Team page users can add/remove/update the team members even in the case of a large number of members. The team members are displayed in a tabular view. Each member has a priority assigned to them for CONLL export which can be changed by dragging them across the list.
+![teamMembers](https://user-images.githubusercontent.com/46840490/193060010-3394cccd-93ab-40a8-8479-98155ea8b417.gif)
+
+**Project Configuration**
+
+The Project Configuration itself is a multi-step process. The wizard will guide users through each step and provide information and hints for each step.
+![projectConfiguration](https://user-images.githubusercontent.com/46840490/193033349-534cc2ab-2e5a-4caa-a050-0ee650949b21.gif)
+
+## Clone
+You can create a copy of a project, by using the Clone option. The option to clone the project is also listed in the kebab menu of each project. The cloned project is differentiated as it contains cloned suffix in its project name.
+
+<img class="image image--xl" src="/assets/images/annotation_lab/4.1.0/dashboardCloneImportExport.png" style="width:100%; align:center; box-shadow: 0 3px 6px rgba(0,0,0,0.16), 0 3px 6px rgba(0,0,0,0.23);"/>
+  
+## Export  
+Entire Projects can be exported. The option to export the project is listed in the kebab menu of each project. All project-related items such as tasks, project configuration, project members, task assignments, and comments are included in the export file.
+
+> **NOTE:**  Project export does not contain the model trained in the project as models are independent and not attached to a particular project.
+  
+## Import  
+The project can be imported by uploading the zip in the upload dialog box. When the project is imported back to ALAB, all the item can be seen as it was present when exported. 
+  
+## Project Grouping
+As the number of projects can grow significantly over time, for an easier management and organization of those, the Annotation Lab allows project grouping. As such, it is possible to assign a project to an existing group or to a new group. Each group can be assigned a color which will be used to highlight projects included in that group. Once a project is assigned to a group, the group name proceeds the name of the project. At any time a project can be remove from one group and added to another group.
+
+The list of visible projects can be filtered based on group name, or using the search functionality which applies to both group name and project name. 
+
+<img class="image image--xl" src="/assets/images/annotation_lab/4.1.0/dashboardGroup.png" style="width:100%; align:center; box-shadow: 0 3px 6px rgba(0,0,0,0.16), 0 3px 6px rgba(0,0,0,0.23);"/>
```

Comment:
You can remove the `align:center;` from the style attribute as it is incorrect property field. And instead of `box-shadow: ....` you can add the class `image__shadow` to the class attribute.